test/retry_test.rb

```diff
@@ -381,6 +387,12 @@ def job
         end
         assert_equal 1, ds.size
       end
+
+      it "supports wrapped jobs" do
+        strat, count = handler.__send__(:delay_for, ActiveJob::QueueAdapters::SidekiqAdapter::JobWrapper, 2, StandardError.new, {"wrapped" => "ActiveJobRetry"})
```

Comment:
Use `ActiveJobRetry.to_s` instead so the code works if we refactor the class name.